modAL/batch.py

```diff
@@ -178,7 +180,7 @@ def ranked_batch(classifier: Union[BaseLearner, BaseCommittee],
         instance_index_ranking.append(instance_index)
 
     # Return numpy array, not a list.
-    return np.array(instance_index_ranking)
+    return np.array(instance_index_ranking), uncertainty_scores[np.array(instance_index_ranking)]
```

Comment:
I don't exactly remember what we discussed, but returning the scores should be an option instead of always having it. So, this might be solved by adding a return_scores: bool argument to the function. This applies to all query strategies.